MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java

```diff
@@ -1143,5 +1153,16 @@ private void initializeModels() {
       new Intent(MainActivity.this, SimpleMapViewActivityKotlin.class),
       null,
       R.string.activity_basic_mapbox_kotlin_url, true, BuildConfig.MIN_SDK_VERSION));
+
+    if (isChinaBuild) {
```

Comment:
Same as before, instead of using a condition check here, we could have two different versions of `MainActivity.java`.